app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,44 @@
-p To be implemented later
+= page_header
+
+div.col-sm-12.col-md-6
+  div.leaderboard-header
+    h4 = t('.level_header')
+  table.table.leaderboard-level
+    tbody
+      - course_users_points = @course_users.ordered_by_experience_points.take(display_user_count)
+      - course_users_points.each.with_index(1) do |course_user, index|
+        = content_tag_for(:tr, course_user, class: ['course-user'])
+          td.rank
+            h3 = index
+          td.user-picture
+            = link_to_course_user(course_user) do
+              = display_user_image(course_user.user)
+          td.user-profile
+            div
+              = link_to_course_user(course_user) do
+                = format_inline_text(course_user.name)
+            div
+              = t('.level', level_number: @course.level_number_for(course_user.experience_points))
```

Comment:
ooops. sorry, my goldfish memory strikes again.

Maybe since this commit is an optimisation, until we find a good way of doing this, let's just stick with the clean way, and then layer the optimisation on top when we have nailed it?